Formula/diff-so-fancy.rb

```diff
@@ -1,17 +1,23 @@
 class DiffSoFancy < Formula
   desc "Good-lookin' diffs with diff-highlight and more"
   homepage "https://github.com/so-fancy/diff-so-fancy"
-  url "https://github.com/so-fancy/diff-so-fancy/archive/v0.6.3.tar.gz"
-  sha256 "b56213f08e5f1de1b0529d1a9a62024913ad2f12e043f9818f8cfcf00bed55c4"
+  url "https://github.com/so-fancy/diff-so-fancy/archive/v0.7.1.tar.gz"
+  sha256 "dcf795df0b398f393215d78679f34427e65aa263be3f4016e7e618706b1b4049"
 
   bottle :unneeded
 
   def install
-    prefix.install Dir["third_party", "libs", "diff-so-fancy"]
+    # temporary fix until upstream uses a directory other
+    # than lib for the perl script.
+    inreplace "diff-so-fancy", "/lib/", "/libexec/"
+    prefix.install "lib" => "libexec"
+
+    prefix.install Dir["third_party", "diff-so-fancy"]
     bin.install_symlink prefix/"diff-so-fancy"
   end
 
   test do
+    ENV["TERM"] = "xterm"
```

Comment:
Nope, this is just a way of setting the terminal value on Jenkins to something that convinces the tool to output in a sane format.